resources/metadata/fortify.yaml

```diff
@@ -245,11 +245,26 @@ spec:
       default: false
     - name: translate
       type: string
-      description: JSON string of list of maps with required key `'src'`, and optional keys `'exclude'`, `'libDirs'`, `'aspnetcore'`, and `'dotNetCoreVersion'`
+      description: "Options for translate phase of Fortify. Most likely, you do not need to set this parameter. See src, exclude. If `'src'` and `'exclude'` are set they are automatically used. Technical details: It has to be a JSON string of list of maps with required key `'src'`, and optional keys `'exclude'`, `'libDirs'`, `'aspnetcore'`, and `'dotNetCoreVersion'`"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - PARAMETERS
+        - STAGES
+        - STEPS
+    - name: src
+      type: string
+      description: Source directories to scan. Multiple entries are separated by space and wildcards can be used, e.g., `'src/main/resources/**/* src/main/java/**/*'`. If set this will be mixed in to `translate`.
```

Comment:
```suggestion
      description: Source directories to scan. Multiple entries are separated by space and wildcards can be used, e.g., `'src/main/resources/**/* src/main/java/**/*'`. If translate is set, this will ignored.
```

Reply:
I'm not sure if the generator works with backticks because I guess they would need escaping in the generated code, right?

Reply:
Maybe you are right, but the backticks were there before.

Reply:
nah, seems to be fine